org.copyq.copyq.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id": "org.copyq.copyq",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "copyq",
+
+    "tag": "v3.2.0",
+    "commit": "ac9f67ee99caa072e85951ba9b6b6f89c1e0ee39",
+
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--env=XDG_CURRENT_DESKTOP=''"
+    ],
+
+    "modules": [
+        {
+            "name": "copyq",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/hluk/CopyQ.git"
```

Comment:
`tag` and `commit` go here =)

Reply:
Oh, that's why it didn't seem to do anything. :) I changed that from nightly builds on the last moment, strangely it looked like it builds the correct version anyways.